modules/telemetry/events/settings-events.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace VIPWorkflow\Modules\Telemetry\Events;
+
+use VIPWorkflow\Modules\Telemetry\Tracker;
+use VIPWorkflow\VIP_Workflow;
+
+function record_admin_update( string $previous_version, string $new_version, Tracker $tracker ): void {
+	// Get all custom statuses
+	$custom_statuses = VIP_Workflow::instance()->custom_status->get_custom_statuses();
+
+	// Get all posts count
+	$posts_count = wp_count_posts();
```

Comment:
Cool function! I tried this locally with a DB containing ~30k posts, and this was approximately instant.

Reply:
Note that [`wp_count_posts()`](https://developer.wordpress.org/reference/functions/wp_count_posts/) only counts `post`-types by default. We should probably loop through each supported post type and sum counts for each status.

Reply:
Done in a2dc1b5fefe29f51c89f84a7a2093ea4f466d251